pkg-manager/plugin-commands-installation/src/update/getUpdateChoices.ts

```diff
@@ -67,12 +68,13 @@ export function getUpdateChoices (outdatedPkgsOfProjects: OutdatedPackage[], wor
         }
       }
       return {
-        name: renderedTable[i],
+        name: outdatedPkg.name,
+        message: renderedTable[i],
         value: outdatedPkg.name,
       }
     })
 
-    finalChoices.push({ name: depGroup, choices })
+    finalChoices.push({ name: depGroup, choices, disabled: true })
```

Comment:
Disable depGroup choice  here, so that the final output of select dependencies will not contain  unexpected "dependencies" or "devDependencies".